src/api_server/src/lib.rs

```diff
@@ -243,7 +243,7 @@ impl ApiServer {
                 response
             }
             Err(err) => {
-                error!("{}", err);
+                error!("{:?}", err);
```

Comment:
This change uses `:?` for all error types. Why didn't you use `:?` in `SerdeJson`?

Reply:
> This change uses :? for all error types. Why didn't you use :? only in SerdeJson?

We can consider that, I think my justifications for preferring applying to all is based on the following:
* Technically `{}` requires that the display trait be implemented, which is not consistenty derived for all error types, but `Debug` should be.
* It is possible that there are other error types that do not involve JSON payload errors (Serde), and as a result may not be sanitized if we just focussed on `SerdeJson` errors.

Reply:
> It is possible that there are other error types that do not involve JSON payload errors (Serde), and as a result may not be sanitized if we just focussed on SerdeJson errors.

Although I cannot see such error types other than `SerdeJson` in `api_server::parsed_request::Error`, I can agree that such error types might be added in the future.